src/openrct2/windows/EditorObjectSelection.cpp

```diff
@@ -897,8 +897,8 @@ void window_editor_object_selection_mousedown(rct_window *w, rct_widgetindex wid
         }
 
         if (!(gScreenFlags & SCREEN_FLAGS_TRACK_MANAGER)) {
-            dropdown_set_checked(DDIX_FILTER_SELECTED, _FILTER_SELECTED);
-            dropdown_set_checked(DDIX_FILTER_NONSELECTED, _FILTER_NONSELECTED);
+            dropdown_set_checked(DDIX_FILTER_SELECTED, _FILTER_SELECTED != 0);
+            dropdown_set_checked(DDIX_FILTER_NONSELECTED, _FILTER_NONSELECTED != 0);
```

Comment:
This does not look very nice at all. Surely we should just change the define to a bool value.